main.go

```diff
@@ -134,13 +146,17 @@ func generatePreprocessCall(g *protogen.GeneratedFile, varName string, repeated
 
 	if repeated {
 		g.P(`for _, v := range `, varName, `{`)
-		g.P(`if v != nil {`)
-		g.P(`v.Preprocess()`)
-		g.P(`}`)
+		g.P(`  if v != nil {`)
```

Comment:
No need to add extra space, it's all to be run through the go formatter. here and everywere.